hangman.rb

```diff
@@ -0,0 +1,72 @@
+
+
+#array of the words to selected for the game
+hangman_words = ["ruby", "python", "coding", "array", "function", "server", "network", "frontend", "backend", "routing",
+                 "template", "middleware", "library", "algorithm", "efficiency", "iteration", "agile", "debug", "script",
+                 "object", "class", "array", "loop", "computer", "server", "framework", "version", "testing", "deploy", "secure"]
+
+selected_word=hangman_words.sample.chars  #to select word and convert it into character array
```

Comment:
```suggestion
selected_word = hangman_words.sample.chars  #to select word and convert it into character array
```

Give space before & after =

Reply:
Please correct at all places

Reply:
Resolved